src/javascript/EditPanel/EditPanelContent/FormBuilder/FieldSet/Field/SingleField.jsx

```diff
@@ -1,40 +1,33 @@
 import React from 'react';
 import * as PropTypes from 'prop-types';
-import {compose} from '~/utils';
-import {connect, FastField} from 'formik';
+import {Field, useFormikContext} from 'formik';
 import {FieldPropTypes} from '~/FormDefinitions';
 
-export const SingleFieldCmp = ({inputContext, editorContext, field, formik, onChange}) => {
-    const FieldComponent = inputContext.fieldComponent;
+// Todo Try to use FastField here -> fixed dependant fields
 
+export const SingleFieldCmp = ({inputContext, editorContext, field, onChange}) => {
+    const FieldComponent = inputContext.fieldComponent;
+    const formik = useFormikContext();
     return (
-        <FastField shouldUpdate={() => true}>
-            {() => {
-                return (
-                    <FieldComponent field={field}
-                                    id={field.name}
-                                    value={formik.values[field.name]}
-                                    values={formik.values}
-                                    editorContext={editorContext}
-                                    inputContext={inputContext}
-                                    onChange={onChange}
-                    />
-                );
-            }}
-        </FastField>
+        <Field component={FieldComponent}
```

Comment:
FastField not usable here (FastField with `shouldUpdate => true` is useless), simplified syntax